src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -101,19 +106,51 @@ private void SetStorageServiceAndTaskHubClient(ref AzureStorageOrchestrationServ
             }
         }
 
+        private void Initialize(ref AzureStorageOrchestrationService orchestrationService, ref TaskHubClient taskHubClient, string connectionStringKey = null, string taskHubName = null)
+        {
+            CheckAssemblies();
+            SetStorageServiceAndTaskHubClient(ref _orchestrationService, ref _client, connectionStringKey, taskHubName);
+        }
+
+        private void CheckAssemblies()
+        {
+            // Retrieve list of DurableTask.AzureStorage DLL files
+            var assemblyFilePaths = Directory.GetFiles(Environment.CurrentDirectory, $"{DurableAzureStorageExtensionName}", SearchOption.AllDirectories).ToList<string>();
+            
+            if (assemblyFilePaths.Count() == 0)
+            {
+                ColoredConsole.WriteLine(Yellow($"Could not find {DurableAzureStorageExtensionName}. The functions host must be running a" +
+                    $" Durable Functions app in order for Durable Functions CLI commands to work."));
+                return;
+            }
+
+            // Convert them to a sorted list of FileInfo objects. Sort by the last modified time of the enclosing bin folder
+            var assemblyFiles = assemblyFilePaths.Select(assemblyFilePath => new FileInfo(assemblyFilePath)).ToList().OrderBy(file => file.Directory.LastWriteTime);
```

Comment:
Remove `.ToList()`. That's unnecessary overhead.

Also, instead of returning a list of assembly files, why not just put the `.Last()` at the end? It seems having the list in a local variable is not useful.